config/sync/language/da/core.date_format.short.yml

```diff
@@ -1 +1,2 @@
 label: 'Standard kort dato'
+pattern: 'm/d/Y - H:i'
```

Comment:
It's coming from the `dce drush cex`

It is not relevant for this pull request, but I think it's a good idea to keep it. Otherwise, it will just be the same for the next person that exports the config files